### ISSUE TITLE
Configure Fall 2020 equipment via LibCal

### ISSUE THREAD

**Issue by cappadona** [cul-it/mann-wagon#1048]
Details TBD, but at high level:

* remove laptops (all laptops will circ from Olin/Uris)
* add inherited equip from Olin/Uris

**Comment by eeh53** [comment]
Yes, correct. Still high level, because this won't launch until later in the semester. So maybe I can just add text to an alert box on this page that says "equipment loans coming soon; to reserve a laptop, please see Olin & Uris website" and then a link to their equipment page.

**Comment by cappadona** [comment]
@eeh53 Adding an alert sounds great to me. Thanks for clarifying that the equipment circulating from Mann will happen at a later date.

I'm going to remove the [Equipment epic](https://github.com/cul-it/mann-wagon/issues/1038) from this immediate milestone unless anyone objects.

**Comment by cappadona** [comment]
@eeh53 I took an initial pass at the [equipment alert](https://mannlib.cornell.edu/use/technology/equipment).

> One of the alerts covered in Box #1059
